helm-chart/binderhub/files/binderhub_config.py

```diff
@@ -46,10 +46,11 @@ def get_value(key, default=None):
 for section, sub_cfg in get_value("config", {}).items():
     c[section].update(sub_cfg)
 
-if get_value("dind.enabled", False) and get_value("dind.hostSocketDir"):
-    c.BinderHub.build_docker_host = "unix://{}/docker.sock".format(
-        get_value("dind.hostSocketDir")
-    )
+containerBuilderPod = get_value("containerBuilderPod")
+if containerBuilderPod in ["dind", "pink"]:
+    hostSocketDir = get_value(f"{containerBuilderPod}.hostSocketDir")
+    if hostSocketDir:
+        c.BinderHub.build_docker_host = f"unix://{hostSocketDir}/docker.sock"
```

Comment:
```suggestion
        c.KubernetesBuildExecutor.docker_host = f"unix://{hostSocketDir}/docker.sock"
```
Does this work for you?

Reply:
Only if I also set the `build_class` traitlet from the `BinderHub` class to use `KubernetesBuildExecutor`.

Otherwise, in `builder.py` the value will be replaced when the `build` object is created starting on line 455.

Reply:
Let's leave it as is for now then, and come back to it later